pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -145,7 +145,7 @@ export class Modifications extends React.Component {
                 </DataListItemRow>
             </DataListItem>;
         }
-        if (this.props.entries !== null && this.props.entries.length === 0) {
+        if (this.props.entries && this.props.entries.length === 0) {
```

Comment:
Do you mean this way: `this.props.entries?.length === 0`? I thought it was unreliable to compare a potentially `undefined` variable against a number.

Reply:
When you do === you do type and value comparison and it's completely safe to do: 

myvar === 0 as it will fail if it's undefined. 

I think the other coments where for the > and < operators.